### ISSUE TITLE
Add new company: Amazon

### ISSUE THREAD

**Issue by yuri-per** [fikrikarim/companies-with-successful-pivot#29]
Amazon was initially an online bookstore.

It's entire AWS business is result of an attempt to create ideal online shopping basket.

**Comment by fikrikarim** [comment]
I searched a bit on google, but it seems that the plan from the beginning was to actually sell everything. Here's [one source](https://www.fastcompany.com/1842702/amazons-pivot):
> Naturally this doesn’t qualify as a pivot since it was part of Bezos’ grand plan from the get go. As he wrote in a 1997 letter to shareholders: “Our goal is to move quickly to solidify and extend our current position while we begin to pursue the online commerce opportunities in other areas. 

On the [wikipedia](https://en.wikipedia.org/wiki/History_of_Amazon#Online_bookstore_and_IPO) they also mention that book is only one of the product they want to start with:
> After reading a report about the future of the Internet that projected annual web commerce growth at 2,300%, Bezos created a list of 20 products that could be marketed online. He narrowed the list to what he felt were the five most promising products, which included: compact discs, computer hardware, computer software, videos, and books. Bezos finally decided that his new business would sell books online, because of the large worldwide demand for literature, the low unit price for books, and the huge number of titles available in print.

I wanted to include Amazon in this list, but I'm not sure whether we could say that Amazon did pivot?

**Comment by 99px** [comment]
Its just expanded not pivoted, there is still huge book section and reader devices

**Comment by fikrikarim** [comment]
Yes totally agree that they expanded not pivoted. I haven't found any link that could indicate the Amazon pivot.

**Comment by ukch** [comment]
I think a lot of people misunderstand Amazon's core business. A larger portion of their profit is made from selling infrastructure through AWS than through selling physical products. I would class that as a successful pivot!

**Comment by LarsDu** [comment]
Amazon never ever pivoted. It simply kept growing larger and larger encompassing more and more.

If it were a dinky startup you might say they successfully pivoted into AWS, but they never actually left the book market. In fact they now dominate books, good, AND cloud which is insane.
